Simulations/YAML/imgN.yaml

```
@@ -0,0 +1,123 @@
+############# WCU DATA ##############
+
+
+DETLIN_GEO_RAW:
+  do.catg: DETLIN_GEO_RAW
+  mode: "img_n"
+  source:
+    name: flat_field
+    kwargs:
+      temperature: 200
+      amplitude: 0
+      filter_curve: "V"
+      extend: 15
+  properties:
+    dit:
+      - 1.
+    ndit: 1
+    filter_name: "open"
+    catg: "CALIB"
+    tech: "IMAGE,N"
+    type: "DETLIN"
+    tplname: "METIS_img_n_cal_DetLin"
+    nObs: 1
+
+
+N_DISTORTION_RAW:
+  do.catg: N_DISTORTION_RAW
+  mode: "img_n"
+  source:
+    name: pinhole_mask
+    kwargs:
+      dx: 5
+      dy: 5
+  properties:
+    dit: 0.25
+    ndit: 1
+    filter_name: "open"
+    nd_filter_name: "ND_OD4"
+    catg: "CALIB"
+    tech: "IMAGE,N"
+    type: "DISTORTION"
+    tplname: "METIS_img_n_cal_distortion"
+    nObs: 1
+
+
+
+##################### "STANDARDS" (SINGLE SOURCE AT CENTER) ######################
+
+#### EACH IMAGE PAIRED WITH SKY ######
+
+
+######### IMAGING ###############
+
+N_IMAGE_STD_RAW1:
+  do.catg: N_IMAGE_STD_RAW
+  mode: "img_n"
+  source:
+    name: calib_star
+    kwargs:
+      amplitude: 12
+  properties:
+    dit: 0.25
+    ndit: 1
+    filter_name: "N1"
+    catg: "CALIB"
+    tech: "IMAGE,N"
+    type: "STD"
+    tplname: "METIS_img_n_cal_standard"
+    nObs: 1
+
+N_IMAGE_STD_RAW2:
+  do.catg: N_IMAGE_SKY_RAW
```

if this is a SKY raw, shouldn't it be called that?
```suggestion
N_IMAGE_SKY_RAW2:
  do.catg: N_IMAGE_SKY_RAW
```

---

Or is this scheme such that some code understands that these two are created with the same template?

---

The first line is simply a unique label for the dictionary read in from the YAML file. We originally used the do.catg, but that's not necessary unique in a set of simulations, so the label is generally close to the do.catg.